### DISCUSSION TITLE
为什么我更新到了1.11.0却还让我安装1.8.0？

### DISCUSSION THREAD

**Discussion by shenguojia123** [share121/inter-knot#2056] [其他]
![image](https://github.com/user-attachments/assets/79341fdb-5b5c-4974-ac84-72ad2e5da7f7)
![image](https://github.com/user-attachments/assets/e84fc854-4d8b-43d2-9b90-182753a2f842)
@share121

**Comment by Dave-12138** [comment]
github pages加了pwa之后就会出现这种上古缓存，简言之， **☝🤓Github干的**
建议使用[绳网.top](https://www.xn--5o0anh.top/)这个域名访问绳网

**Reply by shenguojia123** [reply]
好的，谢谢
